utils/githooks/pre-commit.d/72-pylint.sh

```diff
@@ -16,6 +16,6 @@ if [ -f utils/cq/daos_pylint.py ]; then
             git diff HEAD --name-only | ./utils/cq/daos_pylint.py --files-from-stdin
     else
             echo "  Checking against branch ${TARGET}"
-            git diff "$TARGET"... --name-only | ./utils/cq/daos_pylint.py --files-from-stdin
+            git diff "$TARGET" --name-only | ./utils/cq/daos_pylint.py --files-from-stdin
```

Comment:
This will diff against the branch, rather than against the point where the branch and the PR diverged so it'll check different things.

As master is now clean however then this shouldn't bring in un-related changes and will actually be testing what would land so is probably preferable at this point.  Another change in behaviour between having a clean tree and maintaining a tree with known errors.

Reply:
The previous behavior and current behavior will diff against the point where the branches diverged because:
https://github.com/daos-stack/daos/blob/3129ac8f186e14c4c493de60665ed5f6f1c0a68d/utils/githooks/find_base.sh#L46


Removing the `...` just makes the diff include staged and unstaged changes as well